### PR TITLE
 #18451349 user add products to their cart

### PR DESCRIPTION
**Pull request by Patienceineza**
#### What does this PR do?

this PR is for user adding products to their carts

#### Description of Task to be completed?

this task is to create a route so the current logged in user can add product of their choice to cart
-user should specify the quantity of the product he wants 
- if the user request quantities that are greater than those in stock
- user should get message that requested quantities are unavaliable
- user can also update the cart by adding new quantity to purchase and the cart will be updated
- user can also  delete a certain product from his/her cart

#### How should this be manually tested?

-first clone repo,run `npm run migrate:all`
-login and put token in bearer
- use this end point `/cart/productId`  with post to add product to your cart
-  use `/cart/productId` with delete to remove product from your cart

#### Any background context you want to provide?

n/a

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184581349

#### Screenshots (if appropriate)
Screenshot 2023-03-22 at 18 32 13

Screenshot 2023-03-22 at 18 34 25

#### Questions:

n/a

### CODE REVIEW

**Review by bencyubahiro77 — APPROVED** [review]
check for failing migration with circleCi and you are good to go

**Review by blackd44 — APPROVED** [review]
LG

**Review by nshutijonathan — APPROVED** [review]
- What are the relevant pivotal tracker/Trello stories
- Also add a screenshot of your postman or swagger
LGTM

**Review by edwinnambaje — APPROVED** [review]
LG

**Review by INValentin — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -450,4 +450,65 @@ you will get this response
   ]
 }
 
-```
\ No newline at end of file
+```
+
+## adding product to your wishlist
```
Comment (UmuhireJessie) [inline README.md]: "Adding product to wishlist again?"
Correct this title

**Review by UmuhireJessie — CHANGES_REQUESTED** [review]
See my comment

**Review by UmuhireJessie — APPROVED** [review]

**Pull request merged**